settings.gradle

```diff
@@ -23,6 +23,14 @@ include 'spek-extension:subject:jvm'
 include 'spek-extension:data-driven:common'
 include 'spek-extension:data-driven:jvm'
 
+include 'spek-ide-plugin:interop-common'
+include 'spek-ide-plugin:interop-jvm'
+
+include 'spek-ide-plugin:intellij-common'
+include 'spek-ide-plugin:intellij-jvm'
+include 'spek-ide-plugin:intellij-idea'
+include 'spek-ide-plugin:intellij-studio'
```

Comment:
Sorry, but why do we need separate modules for IDEA and Studio? The platform is the same. Are there any differences?

Reply:
Unfortunately, we need a separate build for Android Studio. There's a bit of hacking needed to properly support it.

Reply:
Hmm, can you elaborate about this please? Just for my personal curiosity ;-) I see that the Kotlin plugin has different versions as well, never got why it is doing that.

Reply:
Pretty sure you are familiar that AS use gradle for building the sources aka "Gradle-aware Make". To get that working with Spek's `RunConfiguration`, you have either to extend one of it's built-in `RunConfiguration` classes or be one of the test frameworks it supports: JUnit, TestNG, Specs2, etc ... I can't use any existing classes and obviously, we don't have first class support.  You also need to implement a marker interface, which is only available in AS.  Somewhere in AS there is a check like:

```
val type = getRunConfiguration(configuration)

if (configuration is MarkerInterface) {
  if (configuration is SomeRunConfigurationSubclass || isTestConfigurationType(type)) {
     // add gradle aware make
  }
}
```

The problem is `isTestConfigurationType` checks against a hard coded list of strings (JUnit, TestNG, Specs2...). I had to trick AS that it's running a `Specs2` configuration, otherwise you have to run gradle manually after making changes to your code.

Reply:
Thanks! That sounds awful though :-( Let’s hope it will work and will not break on every AS update.